src/proto/Ethereum.proto

```diff
@@ -128,6 +131,27 @@ message SigningInput {
 
     // The payload transaction
     Transaction transaction = 10;
+
+    // [EIP4337] Entry point contract address
+    string entry_point = 11;
+
+    // [EIP4337] Account factory contract address
+    string account_factory = 12;
+
+    // [EIP4337] Account logic contract address
+    string account_logic = 13;
+
+    // [EIP4337] Public address of the account signer
+    string owner = 14;
+
+    // [EIP4337] Whether the smart contract for this address has already been deployed
+    bool is_account_deployed = 15;
+
+    // [EIP4337] The amount of gas to pay for to compensate the bundler for pre-verification execution and calldata
+    bytes pre_verification_gas = 16;
+
+    // [EIP4337] The amount of gas to allocate for the verification step
+    bytes verification_gas_limit = 17;
```

Comment:
can you move these to a sub message (we can call it `UserOperation` ) under `Transaction`?